snippets/php/php.json

```diff
@@ -25,10 +25,26 @@
     "function __construct": {
         "prefix": "con",
         "body": [
-            "${1:public} function __construct(${2:${3:Type} \\$${4:var}${5: = ${6:null}}}) {",
-            "\t\\$this->${4:var} = \\$${4:var};$0",
+            "public function __construct(${1: conparg}) {",
```

Comment:
my idea here was to reuse the arguments, in case I want to add more I just use `conparg` snippet.
let me know what you think :)